### PR TITLE
Update release docs

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,11 +35,22 @@ npm run update-schemas
...
+   ```
+2. Push the tag to GitHub and trigger the publish workflow that will publish it on NPM with:
+
+   ```
+   git push [remote] vX.Y.Z
+   ```
+
+3. Unless you are on `master`, create a new branch and push it to GitHub so that the tagged commit belongs to a branch of this repo with:
+   ```
+   git checkout -b X.Y.Z
```
Comment (lf94) [inline README.md:50]: They can do `git tag X.Y.Z` and then `git push X.Y.Z` - creating a branch like that won't create a tag on GitHub automatically AFAIK, at least that was the case in GitLab.

**Review by GestioOp — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nop33) [reply README.md:50]: But in step 1 `npm version` will create the `vX.Y.Z` tag and step 2 will push it on GitHub.

Comment (lf94) [reply README.md:50]: Ah right, in that case, you can simply `git push vX.Y.Z`

Comment (nop33) [reply README.md:50]: https://docs.npmjs.com/cli/v7/commands/npm-version#description
> If run in a git repo, it will also create a version commit and tag.

Comment (lf94) [reply README.md:50]: It will push the reference

Comment (nop33) [reply README.md:50]: > Ah right, in that case, you can simply git push vX.Y.Z

That's what step 2 is...

Comment (nop33) [reply README.md:50]: Your comment is on step 3, right? Is there anything wrong?

Comment (lf94) [reply README.md:50]: After testing I understand the problem better now. Your change is perfect :ok_hand:

**Pull request merged**